### PR TITLE
Add ubuntu 24.04 for Proxmox

### PR DESCRIPTION
**Pull request by alekitto**
## Change description
Add support for Ubuntu 24.04 for Proxmox

## Related issues

- #1406

### CODE REVIEW

**Comment by mboersma** [comment]
/retest

Timeout building ova target, looks like a flake.

File: images/capi/ansible/roles/providers/tasks/proxmox.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 The Kubernetes Authors.
```
Comment (mboersma) [inline images/capi/ansible/roles/providers/tasks/proxmox.yml:1]: This should say "Copyright 2024" since it's a new file.

**Review by mboersma — APPROVED** [review]
/lgtm
/assign @drew-viles 

AFAICT this looks good, although I can't test it directly.

File: images/capi/ansible/roles/providers/tasks/proxmox.yml (hunk 1 above)
Comment (alekitto) [reply images/capi/ansible/roles/providers/tasks/proxmox.yml]: Done :+1:

**Comment by drew-viles** [comment]
/lgtm 
/approve

I'm in the same boat though. It looks logically good to me but I cannot test it

**Pull request merged**